static/scss/header.scss

```diff
@@ -0,0 +1,61 @@
+// sass-lint:disable mixins-before-declarations
+
+.hero-content {
+  position: relative;
+}
+
+.hero-title {
+  background: rgba(black, 0.83);
+  max-width: 580px;
+
+  @include media-breakpoint-down(sm) {
+    max-width: 255px;
+  }
+
+  h1 {
+    color: white;
+    font-weight: 600;
+    font-family: Montserrat;
+    letter-spacing: 0;
+    font-size: 48px;
+    line-height: 48px;
+    padding: 40px 25px 12px 40px;
```

Comment:
Is Montserrat the font that we should be using for the entire site? If so, please update the default "Muli" font in all the relevant places. In other words, if that is the font we're going with, _all_ instances of "Muli" should be replaced with "Montserrat"

Reply:
We should be Montserrat everywhere (except for the header logo, which will need to be an image)